CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeOutput jokeOutput;
+        private IJokeService jokeService;
+
+        public IJokeOutput JokeOutput { 
+            get => jokeOutput; 
+            private set => jokeOutput = value ?? throw new ArgumentNullException(jokeOutput + "is null");

```

Comment:
The `ArgumentNullException` constructor that takes in a string, expect the string to the the parameter name, not a message.
```suggestion
            private set => jokeOutput = value ?? throw new ArgumentNullException(nameof(value));
```